tests/dummy/app/controllers/skip.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+
+export default class SkipController extends Controller {
+  @action
+  auth() {
+    window.grecaptcha.execute();
+  }
+
+  @action
+  forceReset() {
+    this.gRecaptcha.resetReCaptcha();
```

Comment:
We don't have `resetReCaptcha` method it's `reset` now @brunoocasali

Reply:
cool! changed it :D